### ISSUE TITLE
app: simplify webpack config by removing `output.publicPath`

### ISSUE THREAD

**Issue by x1B** [LaxarJS/generator-laxarjs2#13]
Setting a `publicPath` complicates adjustments to various deployments unnecessarily.

_Edit:_ we should avoid an _absolute_ publicPath if possible. A working relative path would be OK, as it does not impact deployment.

**Comment by jonaswjs** [comment]
When using two widgets with different images but with the same name, webpack is loading only one of them.
Solution is to add the path to the loaded files:

```javascript
{  // load styles, images and fonts with the file-loader
   // (out-of-bundle in build/assets/)
   test: /\.(gif|jpe?g|png|ttf|woff2?|svg|eot|otf)(\?.*)?$/,
   loader: 'file-loader',
   options: {
      name: env.production ? 'assets/[name]-[sha1:hash:8].[ext]' : 'assets/[path]-[name].[ext]'
   }
}
```

https://github.com/LaxarJS/generator-laxarjs2/blob/v2.0.0-rc.1/generators/app/templates/webpack.config.js#L52

**Comment by alex3683** [comment]
Fixed on `master` (`v2.0.0`).
